pyEpiabm/pyEpiabm/property/place_foi.py

```diff
@@ -74,7 +74,22 @@ def place_susc(place, infector, infectee,
             Susceptibility parameter of place
 
         """
-        return 1.0
+        place_susc = 1.0
+        place_idx = place.place_type.value - 1
+        if infector.microcell.distancing_start_time is not None:
+            if infector.distancing_enhanced is True:
+                distancing = Parameters.instance().\
+                             intervention_params[
+                             'social_distancing'][
+                             'distancing_place_enhanced_susc'][place_idx]
+            else:
+                distancing = Parameters.instance().\
+                             intervention_params[
+                             'social_distancing'][
+                             'distancing_place_susc'][place_idx]
+        else:
+            distancing = 1
+        return place_susc * distancing
```

Comment:
```suggestion
        if infector.microcell.distancing_start_time is not None:
            if infector.distancing_enhanced is True:
                place_susc *= Parameters.instance().\
                              intervention_params[
                              'social_distancing'][
                              'distancing_place_enhanced_susc'][place_idx]
            else:
                place_susc *= Parameters.instance().\
                              intervention_params[
                              'social_distancing'][
                              'distancing_place_susc'][place_idx]
        return place_susc
```

This is slightly tidier I think (in general good to avoid cluttering the namespace if possible)

Reply:
Done